memos/umbrel-app.yml

```diff
@@ -0,0 +1,53 @@
+manifestVersion: 1
+id: memos
+category: files
+name: Memos
+version: "0.21"
+tagline: Easily capture and share your great thoughts.
+description: >-
+  A privacy-first, lightweight note-taking service.
+  Easily capture and share your great thoughts.
+
+  Open source and free forever. Embrace a future where creativity knows no boundaries with our open-source solution – free today, tomorrow, and always.
+
+  Self-hosting with Docker in just seconds. Enjoy the flexibility, scalability, and ease of setup that Docker provides, allowing you to have full control over your data and privacy.
+
+  Pure text with added Markdown support. Say goodbye to the overwhelming mental burden of rich formatting and embrace a minimalist approach.
+
+  Customize and share your notes effortlessly. With our intuitive sharing features, you can easily collaborate and distribute your notes with others.
+
+  RESTful API for third-party services. Embrace the power of integration and unleash new possibilities with our RESTful API support.
```

Comment:
Creating yaml that renders nicely in the app store UI is a bit finicky. One newline between paragraphs places each paragraph on a separate line, but with no whitespace between. Like this:

<img width="846" alt="image" src="https://github.com/getumbrel/umbrel-apps/assets/85373263/af7cce8c-e6a2-49f0-8591-f633952cdf38">

To get whitespace between each paragraph you can add **2** newlines between each paragraph. Alternatively, you can make each of the paragraphs a bullet point with a single newline between each to render nice bullet points like this:

```
description: >-
  A privacy-first, lightweight note-taking service.
  Easily capture and share your great thoughts.
  
  - Open source and free forever. Embrace a future where creativity knows no boundaries with our open-source solution – free today, tomorrow, and always.
  
  - Self-hosting with Docker in just seconds. Enjoy the flexibility, scalability, and ease of setup that Docker provides, allowing you to have full control over your data and privacy.
  
  - etc...
```

<img width="846" alt="image" src="https://github.com/getumbrel/umbrel-apps/assets/85373263/d002ef73-23c3-43c0-90f5-7f00f45cc8c0">

Reply:
I used bullet points. There could be an issue with the `-` character, but I checked that it parses.